docker-compose.yml

```diff
@@ -11,6 +11,22 @@ services:
     volumes:
       - enterprise_access_mysql57:/var/lib/mysql
 
+  mysql80:
+    command: mysqld --character-set-server=utf8 --collation-server=utf8_general_ci
+    container_name: enterprise_access.mysql80
+    environment:
+      MYSQL_ROOT_PASSWORD: ""
+      MYSQL_ALLOW_EMPTY_PASSWORD: "yes"
+    # Oracle-packaged version includes a `linux/arm64/v8` version, needed for
+    # machines with Apple Silicon CPUs (Mac M1, M2)
+    image: mysql:8.0.33-oracle
+    networks:
+      - devstack_default
+    ports:
+      - "3406:3306"
```

Comment:
You know, we should actually just not bind this here or in enterprise-subsidy, or choose a different host port for each.  Bringing containers up as-is results in "Bind for 0.0.0.0:3406 failed: port is already allocated".  I think I only bound a port in the enterprise-subsidy PR because I was copying some prior art from devstack.